transformers.py

```diff
@@ -34,4 +35,28 @@ def normalise(string):
         this is-a string @swell & so is th1s => thisisastringswellsoisths
     """
     return sub(r"[\W_]+", "", string)
-        
\ No newline at end of file
+
+
+def unit_converter(value, numerator_from, numerator_to, denominator_from=1, denominator_to=1):
+    """
+        unit_converter
+
+    Convert a number into a specified unit type from a specified unit type.
+    
+    Args:
+        value: (int, float) to be converted
+        numerator_from: string that represents the numerator units the data is currently in.
+        numerator_to: string of the desired numerator unit.
+        denominator_from: string of the denominator unit. By default, it is set to 1.
+        denominator_to: string of the denominator unit to be converted to. By default, it is set to 1.
+    
+    Returns:
+        Number that has had unit conversion. For example:
+        10 [m3/s] = 10000/60 [l/min]
+    """
+    try:
+        config_parameters.conversion_dict[(numerator_from, numerator_to)] * value / (config_parameters.conversion_dict[(denominator_from,denominator_to)])
```

Comment:
I noticed you had the default value in the conversion dict to account for this, but a better way of going about it may be to instead just do an if statement to check if we'd be using denominator_from and denominator_to and do the calculation based on whether we are or not.